frontend/pages/signup.js

```diff
@@ -22,9 +23,11 @@ import AddInfo from "./../src/components/signup/AddInfo";
 
 export default function Signup() {
   const { ReactGA } = useContext(UserContext);
+  const router = useRouter();
+  const email = router.query.email;
```

Comment:
Will `.query` always be defined? Want to minimize the chance for `ReferenceError`.